tests/smoke/smoke002-listens-for-complete-event.html

```diff
@@ -0,0 +1,17 @@
+<html>
+    <head>
+        <title>Injectorjs - Injector Complete Event</title>
+        <script src="https://code.jquery.com/jquery-1.12.3.min.js"></script>
+        <script>
+            document.addEventListener(
+                'injectorReady',
+                function (e) {
+                    document.addEventListener("DOMContentLoaded", function(event) {
+                        document.write("<div id='injector-success'>Injector Ready Captured And Processed</div>");
+                    });
+                });
+        </script>
+        <script src="../../src/injector.js"></script>
```

Comment:
It broke for two reasons:

1) The DOM was already loaded before the injector was ready.
2) Document write gets ignored for some reason.

I made a few changes that fixed the issue.